### ISSUE TITLE
[Bug]: Mobile view of Homepage breaks on Mobile

### ISSUE THREAD

**Issue by tobySolutions** [FrancescoXX/free-Web3-resources#385]
### Description

When I go to the homepage of the freeweb3resources.com site on a mobile phone, there are some major text structure breaks happening there. 

If these can be fixed, it'll really help the site's styling to scale properly and be better.

### To Reproduce

1. Go to 'https://freeweb3resources.com' on a mobile phone.
2. Scroll down to the call-to-action part of the site
3. See the styling issues that needs fixing

### Anything else?

![Screenshot from 2023-02-03 11-42-03](https://user-images.githubusercontent.com/96334363/216584923-0cc91cea-47e3-4a4c-8d5e-15a418b24c34.png)

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/FrancescoXX/4c-site/blob/main/docs/CODE_OF_CONDUCT.md)

**Comment by lambeboluwatife** [comment]
@tobySolutions I've fixed this but I'm unable to run the site locally so I can see the changes I made before pushing to github.

**Comment by tobySolutions** [comment]
> @tobySolutions I've fixed this but I'm unable to run the site locally so I can see the changes I made before pushing to github.

You're unable to run the site locally??

**Comment by tobySolutions** [comment]
What error are you getting @lambeboluwatife?

**Comment by lambeboluwatife** [comment]
> What error are you getting @lambeboluwatife?

according to the readme, to run it, you have to run yarn i. But it's giving me "cannot be loaded because running scripts is disabled on this system"

**Comment by tobySolutions** [comment]
No, you can use npm for the project. You don't need to use Yarn.

**Comment by lambeboluwatife** [comment]
> No, you can use npm for the project. You don't need to use Yarn.

really, I'll try it out.
Thanks

**Comment by tobySolutions** [comment]
Still working on this @lambeboluwatife?

**Comment by SoumyadeepOSD** [comment]
Hey @tobySolutions  can I work on it?

**Comment by tobySolutions** [comment]
Sure thing @SoumyadeepOSD!! Let me assign you.

**Comment by lambeboluwatife** [comment]
> Still working on this @lambeboluwatife?

nope. But @SoumyadeepOSD can take over.

Sorry about that, I'm kinda busy with something.

**Comment by tobySolutions** [comment]
Awesome! You can take over this @SoumyadeepOSD!

**Comment by tobySolutions** [comment]
Any updates @SoumyadeepOSD?

**Comment by SoumyadeepOSD** [comment]
> Any updates @SoumyadeepOSD?

@tobySolutions  most probably today I'll create a PR 😁

**Comment by tobySolutions** [comment]
Awesome! Thank you so much @SoumyadeepOSD! Great job.

**Comment by SoumyadeepOSD** [comment]
Hey @tobySolutions  I've created a PR right now about the issue #394 and I think it also solves this issue #385 
You can see here👇

![image](https://user-images.githubusercontent.com/115442240/218652857-2c2e26f5-fcdd-442a-853b-1e82c7acbc0c.png)

**Comment by tobySolutions** [comment]
Great job @SoumyadeepOSD!! Let me check it out.

**Comment by tobySolutions** [comment]
Fixed by #394
